src/components/input-field/examples/input-field-textarea.tsx

```diff
@@ -28,6 +28,7 @@ export class InputFieldTextareaExample {
                 label="Text Field"
                 type="textarea"
                 helperText="Please enter a useful message!"
+                placeholder="Comment text"
```

Comment:
"Comment text" is a little bit confusing as a placeholder. Ideally, we would have something that makes sense as a placeholder, but I can't really think of anything. Perhaps something like `Write a message here…`?

Otherwise, something like "You can put a placeholder here…" would at least make the intention a little clearer. Although I promise someone is going to copy and paste that into their production add-on… 🙈